src/index.ts

```diff
@@ -6,6 +6,12 @@ export * from "./actions/public/bitcoin-tunnel-manager.js";
 export * from "./actions/public/simple-bitcoin-vault.js";
 export * from "./actions/public/simple-bitcoin-vault-state.js";
 export * from "./actions/public/get-btc-finality-by-block-hash.js";
+export { bitcoinTunnelManagerAddresses } from "./contracts/bitcoin-tunnel-manager.js";
+export {
+  encodeChallengeWithdrawal,
+  encodeConfirmDeposit,
+  encodeInitiateWithdrawal,
+} from "./actions/public/bitcoin-tunnel-encoders.js";
```

Comment:
let's move this a few lines above to keep the export sorted by path

Reply:
Don't we have `import/sort` enabled?

Reply:
@gabmontes we discussed the other day... It is not part of [typescript config](https://github.com/bloq/eslint-config-bloq/blob/master/typescript.js). It is enabled on [next config](https://github.com/bloq/eslint-config-bloq/blob/65abc5241a293b915f5d1e83a8004a3394ebfcc3/next.js#L15) though.

Perhaps we could now add the `esm` config (though I would do so in another PR)